allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureLabelsTest.java

```diff
@@ -0,0 +1,58 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.LabelAnnotation;
+import io.qameta.allure.Owner;
+import io.qameta.allure.Story;
+import org.junit.jupiter.api.Test;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+public class AllureLabelsTest {
```

Comment:
Seems like this test lacks a lot of labels... 

<img width="772" alt="image" src="https://user-images.githubusercontent.com/1961118/112134668-a3f57700-8bd5-11eb-8fe3-c397f68f372f.png">

This topic is a bit hard for me to review... Because I don't understand the overall concept of allure labels, like how they are structured, which levels exist, etc. But this also can indicate, that it might be good to implement this example in the way, that it will become more clear to the end user which test-snippets might be useful in which contexts of his daily activities...

Reply:
So... Let's review all labels, group them somehow by meaning, and provide corresponding test-snippets, also in some structural way... mabye even having one test class for labels - is not enough, this is pretty big topic in allure, so maybe we should consider create a separate package with a few test classes for this case... though we will break consistency a bit...